src/clothing_layer.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#ifndef CLOTHING_LAYER_H
+#define CLOTHING_LAYER_H
+
+#include "color.h"
+#include "item.h"
```

Comment:
Would be nice to define the `layer_level` in a new header (not in "item.h") so you don't need to include the whole "item.h" file here.